Fails to compile with current nightly compiler

Compiling with the current nightly (```nightly-x86_64-unknown-linux-gnu updated - rustc 1.27.0-nightly (e5f80f2a4 2018-05-09)```) yields:

```
error[E0599]: no method named `map_or` found for type `syntax::ast::Symbol` in the current scope
  --> src/lib.rs:50:25
   |
50 |             attr.name().map_or(false, |name|
   |                         ^^^^^^
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -47,8 +47,8 @@ impl<'a, 'cx> Folder for Flamer<'a, 'cx> {
 
     fn fold_item_simple(&mut self, i: Item) -> Item {
         fn is_flame_annotation(attr: &Attribute) -> bool {
-            attr.name().map_or(false, |name|
-                    name == "flame" || name == "noflame")
+            let name = attr.name();
+            name == "flame" || name == "noflame"
         }
         // don't double-flame nested annotations
         if i.attrs.iter().any(is_flame_annotation) { return i; }

